docs/user-tutorial/benchmarks/micro-benchmarks.md

```diff
@@ -292,16 +292,16 @@ Support the following traffic patterns:
 
 #### Metrics
 
-| Name                                   | Unit             | Description                                                 |
-|----------------------------------------|------------------|-------------------------------------------------------------|
-| nccl-bw/${operation}_${msg_size}_time  | time (us)        | NCCL operation lantency with given message size.            |
-| nccl-bw/${operation}_${msg_size}_algbw | bandwidth (GB/s) | NCCL operation algorithm bandwidth with given message size. |
-| nccl-bw/${operation}_${msg_size}_busbw | bandwidth (GB/s) | NCCL operation bus bandwidth with given message size.       |
-| rccl-bw/${operation}_${msg_size}_time  | time (us)        | RCCL operation lantency with given message size.            |
-| rccl-bw/${operation}_${msg_size}_algbw | bandwidth (GB/s) | RCCL operation algorithm bandwidth with given message size. |
-| rccl-bw/${operation}_${msg_size}_busbw | bandwidth (GB/s) | RCCL operation bus bandwidth with given message size.       |
-
-If mpi mode is enable and traffic pattern is specified, the metrics pattern will change to `nccl-bw/${operation}_${serial_index)_${parallel_index):${msg_size}_time`
+| Name                                                    | Unit             | Description                                                                            |
```

Comment:
can we keep original metric names for out-of-place, just add new in place metrics?

Reply:
I agree. Please don't change the original metric names.

Reply:
Fixed.